README.md

```diff
@@ -1,18 +1,23 @@
 <!--
-SPDX-FileCopyrightText: 2023 German Aerospace Center (DLR), Forschungszentrum Jülich, Helmholtz-Zentrum Dresden-Rossendorf
+SPDX-FileCopyrightText: 2023 German Aerospace Center (DLR), \
+                             Forschungszentrum Jülich, \
+                             Helmholtz-Zentrum Dresden-Rossendorf
 
 SPDX-License-Identifier: CC0-1.0
 -->
 
 # HERMES continuous integration templates
 
-This repository contains templates for continuous integration systems that can be used to publish software automatically.
+This repository contains templates for continuous integration systems
+that can be used to publish software automatically.
 
 ## License
 
-- Continuous integration templates are licensed under [CC0-1.0](https://github.com/hermes-hmc/ci-templates/blob/main/LICENSES/CC0-1.0.txt).
+- Continuous integration templates are licensed under
+  [CC0-1.0](https://github.com/hermes-hmc/ci-templates/blob/main/LICENSES/CC0-1.0.txt).
 
 Please see the [LICENSES/](LICENSES/) directory and the headers in files for details.
 
-Copyright (c) 2023 German Aerospace Center (DLR), Forschungszentrum Jülich, Helmholtz-Zentrum Dresden-Rossendorf
-
+Copyright (c) 2023 German Aerospace Center (DLR),
+                   Forschungszentrum Jülich,
+                   Helmholtz-Zentrum Dresden-Rossendorf
```

Comment:
**nitpick:** I'm not a big fan of synthetic alignment (it has not impact on the rendered result), plus I see no actual reason to break this line up as it is way less than 120 characters. However, this is something that is most probably highly subjective... and in relation to all the very valuable changes you made, I can most probably live with this... :)

Reply:
Yes, this is not about rendering. As far as I understand markdown should be readable without rendering. And therefore typical tools like [markdownlint](https://github.com/DavidAnson/markdownlint)  claims [MD013 - Line length](https://github.com/DavidAnson/markdownlint/blob/main/doc/md013.md): `README.md:2:81 MD013/line-length Line length [Expected: 80; Actual: 122]`.

Maybe I did not explain my proceeding. I wanted just use your workflow/pipelines in my projects. Therefore I tested it with [pfu](https://gitlab.com/dlr-pa/pfu). And this project already has a ci pipeline using pre-commit. And this pipelines breaks using/integrating your pipelines.

Further I expect many people are using pre-commit and other tools with default configuration. So using a larger line-length is no option.

In the end I tried to make it readable as text and also pass the typical rules.